src/facet/inspection/_learner_inspector.py

```diff
@@ -86,7 +92,7 @@ class LearnerInspector(
 
     def __init__(
         self,
-        model: SupervisedLearnerPipelineDF[T_SupervisedLearnerDF],
+        model: T_SupervisedLearnerDF,
```

Comment:
Pure `T_SupervisedLearnerDF` type has a bound on `SupervisedLearnerDF` which would indicate that pipelines are no longer allowed.
I think it would be better to change type to:
```python
Union[SupervisedLearnerPipelineDF[T_SupervisedLearnerDF], T_SupervisedLearnerDF]
```
and also update the existing docstring:
```
:param model: the learner pipeline to inspect
```

What do you think?

Reply:
`SupervisedLearnerPipelineDF` is a subclass of `SupervisedLearnerDF` so we should be ok as-is. It is a special type of pipeline that guarantees the final estimator to be a `SupervisedLearnerDF`, and the pipeline class implements the full API of class `SupervisedLearnerDF`.

Reply:
Ah, sure - I checked now what `SupervisedLearnerPipelineDF` inherits from. 

Then it's fine for me - last nitpick, if you would like to adjust `model`'s docstring (`the learner pipeline to inspect`) that it also accepts pure learners.

Reply:
Well spotted - thanks - I have updated the docstring